Documentation/design-docs/dllmap.md

```diff
@@ -0,0 +1,270 @@
+# Dllmap design document
```

Comment:
How about add a section about how to debug/diagnostic issues related to using dllmap? will it have ETW events or logs for customer to find out the issue?

Reply:
This is something that _will_ be available in the implementation. At this point, I think it might be enough to indicate that there will be a diagnostic mechanism rather than describe implementation. Regarding 'how it will be enabled', I think it should be enough to say that it will be available in retail builds, but required to be enabled prior to application launch? I am not against enabling on the fly either. Would like to hear other opinions on that.